src/components/form/form-button/form-button.component.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import I18n from 'i18n-js';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags/tags';
+import Button from '../../button';
+
+const text = (props) => {
+  const { formButtonName } = props;
+  const defaultValue = formButtonName.charAt(0).toUpperCase() + formButtonName.slice(1);
+  return props[`${formButtonName}Text`] || I18n.t(`actions.${formButtonName}`, { defaultValue });
+};
+
+const propsForButton = (props) => {
+  return ({
+    save: {
+      as: 'primary',
+      disabled: props.saving,
+      ...props.saveButtonProps
+    },
+    cancel: {
+      onClick: props.cancelClick,
+      type: 'button',
+      ...props.cancelButtonProps
+    }
+  });
+};
+
+const addButtonProps = (props) => {
+  const { formButtonName } = props;
+
+  return {
+    ...propsForButton(props)[formButtonName],
+    'data-element': props.formButtonName,
+    ...props
+  };
+};
+
+const FormButton = (props) => {
+  return (
+    <div { ...tagComponent(props.formButtonName, props) }>
+      <Button { ...addButtonProps(props) }>
+        { text(props) }
+      </Button>
+    </div>
+  );
+};
+
+FormButton.propTypes = {
+  formButtonName: PropTypes.string,
```

Comment:
Maybe this could be strengthened like this?
```js
formButtonName: PropTypes.oneOf(['save', 'cancel'])
```

Reply:
Agreed I've added them to the `OptionsHelper`

Reply:
Thanks :white_check_mark: